src/dune_rules/odoc.ml

```diff
@@ -334,6 +338,7 @@ let setup_html sctx (odocl_file : odoc_artifact) ~pkg ~requires =
                ~dir:(Path.build (Paths.html_root ctx))
                (odoc sctx)
                [ A "html"
+                 (*TODO: we may not need these odoc_base_flags for the html part*)
```

Comment:
To be discussed

Reply:
We could use base_flags here, but include_flags is unnecessary.